src/app/3d/qgspointcloud3dsymbolwidget.cpp

```diff
@@ -134,6 +134,13 @@ QgsPointCloud3DSymbolWidget::QgsPointCloud3DSymbolWidget( QgsPointCloudLayer *la
   mClassifiedRenderingLayout->addWidget( mClassifiedRendererWidget );
 
   connect( mClassifiedRendererWidget, &QgsPointCloudClassifiedRendererWidget::widgetChanged, this, &QgsPointCloud3DSymbolWidget::emitChangedSignal );
+
+  mPointSizeSpinBox->setToolTip( tr( "The size of 1 dataset point in pixels" ) );
+  labelPointSize->setToolTip( tr( "The size of 1 dataset point in pixels" ) );
+  mMaxScreenErrorSpinBox->setToolTip( tr( "Represents the size of the smallest chunk to be rendered in pixels, the less the value the more chunks will be rendered" ) );
+  labelMaxScreenError->setToolTip( tr( "Represents the size of the smallest chunk to be rendered in pixels, the less the value the more chunks will be rendered" ) );
+  mPointBudgetSpinBox->setToolTip( tr( "Sets the number of points that will be renderered simultaniously at most" ) );
+  labelPointBudget->setToolTip( tr( "Sets the number of points that will be renderered simultaniously at most" ) );
```

Comment:
I'd suggest something like:

```suggestion
  mPointSizeSpinBox->setToolTip( tr( "The size of each point in pixels" ) );
  mMaxScreenErrorSpinBox->setToolTip( tr( "The size in pixels of the smallest chunk to be rendered.\nRaising this value will result in a less detailed scene which can improve performance" ) );
  mPointBudgetSpinBox->setToolTip( tr( "The maximum number of points that will be rendered simultaneously.\nRaising this value may allow missing chunks to be rendered while lowering it may improve performance" ) );
```